extensions/db/src/test/resources/integration/storedprocedure/stored-procedure-streaming-returning-resultsets-config.xml

```diff
@@ -10,14 +10,14 @@
             <db:sql>{ call getSplitTestRecords() }</db:sql>
         </db:stored-procedure>
 
-        <set-variable variableName="records" value="#[[]]"/>
+        <set-variable variableName="records" value="#[[]]" mimeType="application/java"/>
```

Comment:
I think since this is going to end up in compatibility, it doesn't even worth to change it. WDYT?